### ISSUE TITLE
Orientation of clipped sequences when copying

### ISSUE THREAD

**Issue by kevfengler227** [igvteam/igv#1487]
When you copy a clipped sequence it is in the orientation of the read, but it may make more sense to be in the orientation of the alignment. I keeping making this mistake. If you copy the clipped sequence from one of these 3 reads you may expect to get a sequence starting in "CGCAG" but only if you select a read in the forward orientation!

![image](https://github.com/igvteam/igv/assets/20604003/6639e40e-373a-43f4-bb1e-2c61db851035)

**Comment by jrobinso** [comment]
Hmm,  actually its copying in the orientation of the alignment now.   I.E. reads aligned to the reverse strand are reverse complemented.   I'm not sure why this is done, I will look into it.    One could argue either way, but the read sequence (as opposed to clipped sequences) is copied without reverse complementing.   I think they should be consistent.

The SAM/BAM/CRAM formats always record the sequence in the forward alignments, and there is a flag to indicate the sequence as recorded is reverse complimented.   I.E. the alignment was to the reverse strand.

**Comment by jrobinso** [comment]
After some review I've concluded the current behavior was a mistake, specifically copying the reverse-complement of the alignment read sequence on reverse strand alignments.   Although this is the actual clipped read sequence, it is not in keeping with SAM/BAM convention nor consistent with "copy read sequence".   I think this happened by using the same code as "blat clipped sequence".   When blatting you do want the actual read sequence,  otherwise the alignment strands will be off.  

So this has been fixed, will be in the next point release which is due out very soon.
